updates/version.yaml

```diff
@@ -44,3 +44,4 @@
 1.2.17: Changes locations of custom fields to secondary tabs instead of the primary Settings area. New menu search ability on adding menu items
 1.2.18: Fixes cache-invalidation issues when RainLab.Translate is not installed. Added Greek & Simplified Chinese translations. Removed deprecated calls. Allowed saving HTML in snippet properties. Added support for the MediaFinder in menu items.
 1.2.19: Catch exception with corrupted menu file.
+1.2.20: Menu name visible in component as a property + menu.referencesGenerated event fired.
```

Comment:
Should be "StaticMenu component now exposes menuName property; added pages.menu.referencesGenerated event.